Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ruby:2.6.5-alpine
+ENV VERSION v0.1.0
+ENV INFO Truemail lightweight rack based web API 🚀
+ENV APP_USER truemail
+ENV APP_HOME /var/lib/truemail-rack
+ENV TMP /var/lib/truemail-rack/tmp
+ENV APP_PORT 9292
```

Comment:
Can be
![image](https://user-images.githubusercontent.com/20876093/75539536-0f950100-5a23-11ea-87b0-413cdfd924fc.png)